libs/algorithms/include/hpx/parallel/algorithms/copy.hpp

```diff
@@ -266,10 +266,10 @@ namespace hpx { namespace parallel { inline namespace v1 {
 
             template <typename ExPolicy, typename InIter, typename Sent,
                 typename OutIter>
-            static util::in_out_result<InIter, OutIter> sequential(
+            static hpx::ranges::in_out_result<InIter, OutIter> sequential(
```

Comment:
I now remember why I left the `in_out_result` type in namespace util: I wanted to avoid mentioning the `namespace hpx::ranges` in the base algorithms. This is not a big deal, but seemed to be the right thing at that time.

Could we leave the type in `util` and just introduce a `using` directive in the ranges namespace to make in conform to the standard?

Reply:
Yes, I can definitely do that.